tests/test_findshift1.py

```diff
@@ -0,0 +1,495 @@
+import os, glob
+import generate_tempfiles
+import numpy as np
+from calcos import findshift1
+
+findshift1_obj = None
+findshift1_obj2 = None
+shift1_dict = {}
+shift1_dict2 = {}
+fp_pxl_shft = {}
+fp_pxl_shft2 = {}
+spectra = {}
+spectra2 = {}
+template = {}
+template2 = {}
+xc_range = []
+xc_range2 = []
+keys = ['FUVA', 'FUVB']
+keys2 = ['NUVA', 'NUVB', 'NUVC']
+
+
+def test_init():
+    # setup
+    file_names = ["temp_wcp_file.fits", "temp_lamptab_file.fits", "temp_lampflash_file.fits"]
+    file_names2 = ["temp_wcp_nuv_file.fits", "temp_lamptab_nuv_file.fits", "temp_lampflash_nuv_file.fits"]
+    hdu_lists = [generate_tempfiles.create_wcp_file(file_names[0]),
+                 generate_tempfiles.create_lamptab_file(file_names[1]),
+                 generate_tempfiles.create_lampflash_file(file_names[2])]
+    hdu_lists2 = [generate_tempfiles.create_wcp_file(file_names2[0], 'NUV'),
+                  generate_tempfiles.create_lamptab_file(file_names2[1], 'NUV'),
+                  generate_tempfiles.create_lampflash_file(file_names2[2], "NUV")]
+    spectra["FUVA"] = hdu_lists[2][1].data['NET'][0] * hdu_lists[2][1].data['EXPTIME'][0]
+    spectra["FUVB"] = hdu_lists[2][1].data['NET'][1] * hdu_lists[2][1].data['EXPTIME'][1]
+
+    spectra2["NUVA"] = hdu_lists2[2][1].data['NET'][0] * hdu_lists2[2][1].data['EXPTIME'][0]
+    spectra2["NUVB"] = hdu_lists2[2][1].data['NET'][1] * hdu_lists2[2][1].data['EXPTIME'][1]
+    spectra2["NUVC"] = hdu_lists2[2][1].data['NET'][2] * hdu_lists2[2][1].data['EXPTIME'][2]
+    # fp_pixel_shift from lamptab file
+    template['FUVA'] = np.ones(16384)
+    template['FUVB'] = np.zeros(16384)
+
+    template2['NUVA'] = np.ones(16384)
+    template2['NUVB'] = np.ones(16384)
+    template2['NUVC'] = np.zeros(16384)
+
+    for seg, seg2 in zip(hdu_lists[1][1].data['SEGMENT'], hdu_lists2[1][1].data['SEGMENT']):
+        fp_pxl_shft[seg] = hdu_lists[1][1].data['FP_PIXEL_SHIFT']
+        fp_pxl_shft2[seg2] = hdu_lists2[1][1].data['FP_PIXEL_SHIFT']
+    initial_offset = 0
+    info = hdu_lists[2][0].header
+    # todo fails due to keyError (X_OFFSET)
```

Comment:
Was this fixed?

Reply:
Yes, I just forgot to delete the todo comment.